### DISCUSSION TITLE
Does windicss support apply syntax?

### DISCUSSION THREAD

**Discussion by pepelele** [windicss/windicss#380] [Q&A]
I'm working on VechaiUI ... It's a tailwindcss plugin and I want to support windicss. But I realized that I write them with apply syntax instead of css-in-js syntax like this https://github.com/vechai/vechaiui/blob/main/packages/core/src/components/button.js. I don't know if there is a way to make it work with the windicss plugin?

**Comment by hannoeru** [comment]
You can use Transform API to transform Tailwind plugin to Windi CSS plugin

https://windicss.org/plugins/transform.html
